src/tests/ftest/util/test_utils_pool.py

```diff
@@ -539,6 +539,47 @@ def check_pool_daos_space(self, s_total=None, s_free=None):
             for index, item in enumerate(val)]
         return self._check_info(checks)
 
+    def check_free_space(self, expected_scm=None, expected_nvme=None,
+                         timeout=30):
```

Comment:
Optional/Minor - FYI max line length is 100
```suggestion
    def check_free_space(self, expected_scm=None, expected_nvme=None, timeout=30):
```